bin/ap_proto

```diff
@@ -0,0 +1,337 @@
+#!/bin/env python
+
+"""
+Script which simulates AP workflow access to L1 datanbase.
+
+It generates approximately realistic result of difference image analysis,
+source-to-object matching, and forced photometry and stores all that in
+a database.
+"""
+
+#--------------------------------
+#  Imports of standard modules --
+#--------------------------------
+from argparse import ArgumentParser
+from datetime import datetime, timedelta
+import logging
+import math
+import numpy
+import os
+import sys
+
+#-----------------------------
+# Imports for other modules --
+#-----------------------------
+from lsst.l1dbproto import generators, l1db, DIA, constants
+from lsst.sphgeom import LonLat, Vector3d, htmIndex, UnitVector3d
+
+#---------------------
+# Local definitions --
+#---------------------
+
+_LOG = logging.getLogger('ap_proto')
+
+def _configLogger(verbosity):
+    """ configure logging based on verbosity level """
+
+    levels = {0: logging.WARNING, 1: logging.INFO, 2: logging.DEBUG}
+    logfmt = "%(asctime)s [%(levelname)s] %(name)s: %(message)s"
+
+    logging.basicConfig(level=levels.get(verbosity, logging.DEBUG), format=logfmt)
+
+_dt_format = ('%Y-%m-%d %H:%M:%S', 'YYYY-MM-DD hh:mm:ss')
+
+def _timeParse(dt_str):
+    """ Convert string to datetime object """
+    dt = datetime.strptime(dt_str, _dt_format[0])
+    return dt
+
+def _isDayTime(dt):
+    """ Returns true if time is not good for observing """
+    return 6 <= dt.hour < 20
+
+def _visitTimes(start_time, interval_sec, count):
+    """
+    Generator for visit times
+    """
```

Comment:
In some functions there are comments in a single line while others are in this format. I like this one better if a standard must be chosen.